native_extensions/emoji_selector/src/lib.rs

```diff
@@ -128,6 +128,30 @@ fn render_selector<'a>(
             emoji_selector.focus();
         "#;
 
+    fn should_display_emoji(emoji: &Emoji) -> bool {
```

Comment:
We could simplify this with the `cfg!` macro into doing something like

```rust
   (cfg!(target_os = "windows") && emoji.unicode_version().major() < 15) || true
```
(or use `cfg!(not(target_os="windows"))` instead of `true` if you run into the lint error) which could probably just be apart of the iterator logic instead in its own function

Reply:
SO much cleaner! I knew there had to be a better way